configuration/rules-blockly-voice-and-multimedia.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: documentation
+title: Rules Blockly - Voice and Multimedia
+---
+
+# Multimedia
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+## Introduction
+
+In general, this section deals with two topics
+
+1) Playing or streaming audio to an audio sink e.g a speaker
+2)  Saying a text via using any Text-to-Speech API (e.g. Google's API)
+
+It should be noted that both functionalities do not just work right out of the box because e.g for (1) the audio sinks need to be installed in openHAB and for (2) e.g. the whole Google API needs to be configured in particular with your own personal google account. Even though (2) is not trivial but after having done that the usage of the functionality with blockly is very easy and worth the effort.
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+{: #blockly-ephemeris-overview}
+
+## Overview of the Voice and Multimedia blocks
+
+![voice-and-multimedia](images/blockly/blockly-voice-and-multimedia.png)
+
+## Setup
+
+* Audio Sinks
+  - Audio devices are automatically added by their bindings as *sinks* in openHAB
+  -  [Reference on how to setup Audio and Audio Sinks](https://www.openhab.org/docs/configuration/multimedia.html)
+* Text-To-Speech
+  - In order to use text-to-speech, you need to install at least one [TTS service](https://www.openhab.org/addons/#voice).
+  - see [openHAB Voice](https://www.openhab.org/docs/configuration/multimedia.html#voice) for more information
+  - in case you only have one TTS service, this becomes your default TTS services. In case you have several TTS services installed, you have to configure one of them as the default which can be done in your [services.cfg ](https://github.com/openhab/openhab-distro/blob/master/distributions/distribution-resources/src/main/resources/runtime/etc/services.cfg#L20). See more on that special case [here](https://community.openhab.org/t/where-to-set-default-tts-voice/15450/2)
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.

Should also work foir list elements when you do not add the next dash for the next list element